File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int ans = 0;
        for(int row=0; row<accounts.length; row++){
            int tempSum = 0;
            for(int col=0; col<accounts[row].length; col++){
                tempSum += accounts[row][col];
            }
            if(tempSum>ans){
                ans = tempSum;
            }
        }
        return ans;
    }
}

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        maxSum = 0
        for i in range(len(accounts)):
            currSum=0
            for j in range(len(accounts[i])):
                currSum+=accounts[i][j]
            maxSum = max(currSum, maxSum)
        return maxSum